samples/mpu/mpu_stack_guard_test/README.rst

```diff
@@ -35,33 +35,40 @@ To build the test with the MPU enabled and the stack guard feature present:
 Sample Output
 =============
 
-With the MPU disabled:
+With the MPU enabled but the stack guard feature disabled:
```

Comment:
since you used the phrase"stack guard feature disabled" here, should line 62 below say "stack guard feature enabled" to be consistent?